content/blog/2019/02/2019-02-08-let-s-celebrate-java-11-support.adoc

```diff
@@ -0,0 +1,113 @@
+
+Starting in June 2018, many events were organized to improve Jenkins code base and add Java 11 support.
+Beyond these events, Core/Plugins maintainers and many other contributors have worked hard to make sure they discover and solve as many issues as possible related to Java 11 support.
+
+The effort to support Java 11 led to the creation of the https://github.com/jenkinsci/jep/blob/master/jep/211/README.adoc[JEP-211: Java 10+ support in Jenkins].
+It also spurred the creation of the https://jenkins.io/sigs/platform[Platform Special Interest Group] to coordinate the Java 11 work and other platform support efforts.
+
+== Celebration
+
+We'd like to take a moment to thank everyone involved in these tasks: code contributors, issue reporters, testers, event planners and attendees and all those in the community who have generously lent their time and support to this effort.
+Thank you all!
+
+Here are some of the contributors who helped with this task (alphabetical order):
+
+Alex Earl,
+Alyssa Tong,
+Baptiste Mathus,
+Carlos Sanchez,
+Daniel Beck,
+David Aldrich,
+Denis Digtyar,
+Devin Nusbaum,
+Emeric Vernat,
+Evaristo Gutierrez,
+Gianpaolo Macario,
+Isabel Vilacides,
+James Howe,
+Jeff Pearce,
+Jeff Thompson,
+Jesse Glick,
+Jonah Graham,
+Kevin Earls,
+Ksenia Nenasheva,
+Kohsuke Kawaguchi,
+Liam Newman,
+Mandy Chung,
+Mark Waite,
+Nicolas De Loof,
+Oleg Nenashev,
+Oliver Gondža,
+Olivier Lamy,
+Olivier Vernin,
+Paul Sandoz,
+Ramón León,
+Sam Van Oort,
+Tobias Getrost,
+Tracy Miranda,
+Ulli Hafner,
+Vincent Latombe,
+Wadeck Follonier,
+
+(We are deeply sorry if we missed anyone in this list.)
+
+== Guidelines
+
+In order to keep it simple, here is how you can start Jenkins on Java 11 using the Docker image.
+You can select a Java 11 based image by suffixing the tag of the image with `-jdk11`.
+If you are upgrading an existing instance please read the link:/doc/administration/requirements/upgrade-java-guidelines/[Upgrading Jenkins Java version from 8 to 11] page before upgrading.
+
+So you can run Jenkins on Java 11 with:
+
+[source, shell]
+----
+docker run -p 50000:50000 -p 8080:8080 jenkins/jenkins:2.164-jdk11
+----
+
+However, and as always, you can still start Jenkins with other methods.
+Please see the more detailed documentation at link:/doc/administration/requirements/jenkins-on-java-11[Running Jenkins on Java 11].
+
+== Developer guidelines
+
+For developers involved in Jenkins development, you can find details on developing and testing Jenkins to run on Java 11 on the link:https://wiki.jenkins.io/display/JENKINS/Java+11+Developer+Guidelines[Java 11 Developer Guidelines].
+
+This resource regroups the modifications which might need to be done in order to validate the compatibility of plugins for Java 11.
+
+== What's next
+
+Even though this is a big achievement, we still have work to do.
+
+Our first priority is adding Java 11 support to link:https://github.com/jenkinsci/jenkinsfile-runner[JenkinsFile Runner] project.
```

Comment:
It is actually done, except some test automation

Reply:
yes, at the time I wrote this I thought it wasn't. Do you want to remove this?

Reply:
I think we could keep it as is until it is announced